code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -147,7 +147,7 @@
 	temp_change = (temperature - current)
 	return temp_change
 
-/mob/living/simple_animal/slime/handle_chemicals_in_body()
+/*/mob/living/simple_animal/slime/handle_chemicals_in_body()
```

Comment:
don't comment out code, just removed it, git stores backups should we ever want to revert.